.github/workflows/azdevops-agent-release.yml

```diff
@@ -1,69 +1,32 @@
-name: Release azure devops agent
+name: Scan/Release azure devops agent
 
 on:
   push:
     tags:
       - "azdevops-agent-*"
-
+    branches:
+      - main
+    paths:
+      - .github/workflows/azdevops-agent-release.yml
+      - infrastructure/images/azure-devops-agent/**
+      - .github/workflows/reusable-image-scan-and-release-ghcr.yaml
```

Comment:
nit: rename to .yml for consistency with the rest.

Reply:
Fixed